pkg/config/log_test.go

```diff
@@ -26,6 +26,9 @@ func TestExtractShortPathFromFullPath(t *testing.T) {
 	assert.Equal(t, "main.go", extractShortPathFromFullPath("main.go"))
 	// process agent
 	assert.Equal(t, "cmd/agent/collector.go", extractShortPathFromFullPath("/home/jenkins/workspace/process-agent-build-ddagent/go/src/github.com/DataDog/datadog-process-agent/cmd/agent/collector.go"))
+	// various possible dependency paths
+	assert.Equal(t, "go.opentelemetry.io/collector@v0.35.0/receiver/otlpreceiver/otlp.go", extractShortPathFromFullPath("/Users/slavek.kabrda/programming/go/pkg/mod/go.opentelemetry.io/collector@v0.35.0/receiver/otlpreceiver/otlp.go"))
+	assert.Equal(t, "go.opentelemetry.io/collector@v0.35.0/receiver/otlpreceiver/otlp.go", extractShortPathFromFullPath("/gomodcache/go.opentelemetry.io/collector@v0.35.0/receiver/otlpreceiver/otlp.go"))
```

Comment:
It would be cool to have a unit test that actually takes the path for some dependency and testes this out with that path (so that we test this with the actual path we get in all our builders and platforms) but I don't know if that would be easy to do